tests/stub/routing.py

```diff
@@ -860,74 +963,33 @@ def work(tx):
         self._readServer1.done()
         self.assertEqual([[1]], sequences)
 
-    def test_should_round_robin_readers_when_reading_using_session_run(self):
```

Comment:
These 2 tests were ported from java driver. I know they are very implementation specific, but we could just make them a bit more lenient and basically check if one of the servers gets used and goes down then the driver uses the other server. However, we may already have such tests.

Reply:
We have some tests that go in that direction. The biggest problem is, how do make sure that the first server the driver contacts will be the broken one? This would require some sort of callback or communication between the servers, I think.

Reply:
Sorry, I did not mean breaking the server. I just expect the server to finish the script and exit, which would leave the second server running. But if you believe there are tests that already cover this case, then it would just be a duplicate :)

Reply:
But why should a driver be allowed to contact the same server twice?